engines/rancher_on_eks/app/views/rancher_on_eks/wrapup/show.html.haml

```diff
@@ -10,6 +10,11 @@
   = markdown(t('engines.rancher_on_eks.wrapup.url', fqdn: @fqdn))
   = markdown(t('engines.rancher_on_eks.wrapup.password', password: @password))
   = markdown(t('engines.rancher_on_eks.wrapup.more_info', region: @region, cluster_name: @cluster_name))
+- elsif @failed
```

Comment:
Okay, so now we have a title and caption if: 
* `failed == true && cleaning_up == false` https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/38b8a60315b33dd62b92cc2202ede0a6bdb9215e/engines/rancher_on_eks/app/views/rancher_on_eks/wrapup/show.html.haml#L1
* `failed == false && cleaning_up == false` https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/38b8a60315b33dd62b92cc2202ede0a6bdb9215e/engines/rancher_on_eks/app/views/rancher_on_eks/wrapup/show.html.haml#L6
* `failed == true && cleaning_up == true` https://github.com/SUSE-Enceladus/suse-rancher-setup/blob/38b8a60315b33dd62b92cc2202ede0a6bdb9215e/engines/rancher_on_eks/app/views/rancher_on_eks/wrapup/show.html.haml#L13

... but no title & caption if `failed == false && cleaning_up == true`. Is that intentional?

Reply:
I do not think so, IIRC we showed title + caption during clean up
but that is not the case in `next` branch (as it is right now)
![image](https://user-images.githubusercontent.com/3913611/213697633-f8b06c42-486a-4902-93d4-fce38ce1768d.png)
I have found that and another UI thing I will address both after this PR, as this one aims to fix issue #121 only